### PR TITLE
New Duo Rule: Duo Admin Bypass Code Viewed

### PR DESCRIPTION
**Pull request by andrea-youwakim**
### Background

<High level overview here>

### Changes

* adding new duo rule to monitor and alert on when an admin views a user's mfa bypass code

### Testing

* 
Screen Shot 2022-12-16 at 3 40 58 PM

### CODE REVIEW

File: rules/duo_rules/duo_admin_bypass_code_viewed.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+AnalysisType: rule
+Description: An administrator viewed the MFA bypass code for a user.
+DisplayName: Duo Admin Bypass Code Viewed
+Enabled: true
+Filename: duo_admin_bypass_code_viewed.py
+Reference: https://duo.com/docs/adminapi
+Runbook: 'Confirm this behavior is authorized: The security of your Duo application is tied to the security of your secret key (skey). Secure it as you would any sensitive credential. You should not share it with unauthorized individuals or email it to anyone under any circumstances!'
```
Comment (edyesed) [inline rules/duo_rules/duo_admin_bypass_code_viewed.yml:7]: This does not need quotes ( though no particular harm in having them )

**Review by edyesed — APPROVED** [review]
Unnecessary single quoting in the Runbook, though it is mostly harmless aside from consistency with other rules

File: rules/duo_rules/duo_admin_bypass_code_viewed.yml (hunk 1 above)
Comment (andrea-youwakim) [reply rules/duo_rules/duo_admin_bypass_code_viewed.yml]: fixed in new commit!

**Comment by jpl5280** [comment]
@andrea-youwakim - Going forward, please paste output of your tests, from the CLI, in the PR description, so we can see the output vs. Panther Console screenshot. Thanks!

**Review by jpl5280 — APPROVED** [review]

**Review by edyesed — APPROVED** [review]
new duo detection for bypass codes, whatever those may be

**Pull request merged**